src/ray/gcs/pb_util.h

```diff
@@ -327,6 +381,34 @@ inline void FillTaskStatusUpdateTime(const ray::rpc::TaskStatus &task_status,
   (*state_updates->mutable_state_ts_ns())[task_status] = timestamp;
 }
 
+/// Fill the rpc::ExportTaskEventData::TaskStateUpdate with the timestamps
+/// according to the status change.
+///
+/// \param task_status The task status.
+/// \param timestamp The timestamp.
+/// \param[out] state_updates The state updates with timestamp to be updated.
+inline void FillExportTaskStatusUpdateTime(
+    const ray::rpc::TaskStatus &task_status,
+    int64_t timestamp,
+    rpc::ExportTaskEventData::TaskStateUpdate *state_updates) {
+  if (task_status == rpc::TaskStatus::NIL) {
+    // Not status change.
+    return;
+  }
+  (*state_updates->mutable_state_ts_ns())[task_status] = timestamp;
+}
+
+/// Convert rpc::TaskLogInfo to rpc::ExportTaskEventData::TaskLogInfo
+inline void TaskLogInfoToExport(const rpc::TaskLogInfo &src,
+                                rpc::ExportTaskEventData::TaskLogInfo *dest) {
+  dest->set_stdout_file(src.stdout_file());
```

Comment:
This function is called with src being `state_update_->task_log_info_.value()` so we probably shouldn't change src because it's a `TaskStatusEvent` class attribute